tools/bootstrap/README.md

```diff
@@ -6,7 +6,7 @@ A [Boxstarter][] script can be used for easy setup of Windows systems with all
 the required prerequisites for Node.js development. This script will install
 the following [Chocolatey] packages:
   * [Git for Windows][] with the `git` and Unix tools added to the `PATH`
-  * [Python 2.x][]
+  * [Python 3.7][] and [legacy Python][]
```

Comment:
Nit: Can we just mention Python 3.x? Whenever there is a new release of Python 3 with minor version bumped, we have to update this document. Moreover, I don't think any of our scripts are using a feature which is available only after a specific version of Python 3.